#!/usr/bin/env python
PACKAGE = "point_cloud_spatial_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_min", double_t, 0, "The minimum value for X", -5, -2, 0)
gen.add("x_max", double_t, 0, "The maximum value for X", 0, 2, 5)
gen.add("y_min", double_t, 0, "The minimum value for Y", -5, -2, 0)
gen.add("y_max", double_t, 0, "The maximum value for Y", 0, 2, 5)
gen.add("z_min", double_t, 0, "The minimum value for Z", 0, 0, 5)
gen.add("z_max", double_t, 0, "The maximum value for Z", 0, 2, 5)

exit(gen.generate(PACKAGE, "point_cloud_spatial_filter", "DynamicConfiguration"))
